src/shaders/describe_shaders

```diff
@@ -49,7 +48,7 @@ style.layers.forEach((layer) => {
 
 function addProgramInfo(layer, programInterface, shaderDefinition) {
     const styleLayer = new StyleLayer(layer);
-    styleLayer.updatePaintTransitions([], {}, { zoom: zoom }, new AnimationLoop(), {});
+    styleLayer.updatePaintTransitions([], {}, { zoom: zoom }, {});
     const configuration = ProgramConfiguration.createDynamic(
         programInterface, styleLayer, zoom);
     const key = `${layer.type}${configuration.cacheKey || ''}`;
```

Comment:
I'd be okay with just removing `describe_shaders` -- it's not part of any regular workflows, and that makes it very likely to go stale

Reply:
👍